### DISCUSSION TITLE
OTEL_TRACES_EXPORTER=none value not recognized

### DISCUSSION THREAD

**Discussion by hballangan-mdsol** [open-telemetry/opentelemetry-java-instrumentation#2326] [Q&A]
Hi! As specified in the docs OTEL_TRACES_EXPORTER=none should be available (I wanted to turn both OTEL_TRACES_EXPORTER and OTEL_METRICS_EXPORTER off on local and travis build), but none is not a recognized value for OTEL_TRACES_EXPORTER. 
![image](https://user-images.githubusercontent.com/70624445/108309961-85bee480-71ed-11eb-9c09-7fc9a69ad115.png)

**Comment by anuraaga** [comment]
Hi @hballangan-mdsol - are you able to add the property `-Dotel.traces.exporter=none`? I think currently there's a problem with the environment variable handling - it will be fixed in 0.17.0 because we have moved the default values / handling of none from the agent to the SDK.
